### DISCUSSION TITLE
Non-additive Models

### DISCUSSION THREAD

**Discussion by DarioS** [PGScatalog/pgsc_calc#313] [Q&A] [answered]
PGS Catalog has incorporated non-additive models from PGP000546 two days ago. Can PGSC Calc use such models correctly?
![image](https://github.com/PGScatalog/pgsc_calc/assets/631218/f81c308a-da07-4bc0-8766-aa6167c526fb)

**Comment by smlmbrt** [comment, marked as answer]
No, not as of yet. It can handle dominant/recessive-coded alleles but not ones where each allele combination has a different weight. I Will create an issue.

**Reply by smlmbrt** [reply]
Noted in: https://github.com/PGScatalog/pgsc_calc/issues/314
